.github/workflows/ci.yml

```diff
@@ -0,0 +1,95 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+
+jobs:
+
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
+    - name: Cache pip
+      uses: actions/cache@v2.1.6
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/dev-requirements.txt') }}
```

Comment:
I think this key is wrong?

The below install step runs:
```
	@pip install -Ur requirements-dev.txt
	@pip install demos/polls
	@pip install demos/chat
	@pip install demos/blog
	@pip install demos/graphql-demo
	@pip install -r demos/moderator_bot/requirements-dev.txt
```
So, I think this is installing from files with some named `requirements-dev.txt` and some named `requirements.txt`. None are named `dev-requirements.txt`. Can we do `**/requirements*.txt` maybe?

Reply:
I think `**/requirements*.txt` will work.